examples/generator-fullstack/server/src/index.ts

```diff
@@ -20,40 +18,40 @@ async function start() {
   app.get('/health', (req, res) => res.sendStatus(200));
 
   const config = await loadConfig({
-    extensions: [() => ({ name: 'generate' })]
+    extensions: [() => ({ name: 'graphback' })]
   });
 
-  const generateConfig = await config!.getDefault().extension('generate');
+  const generateConfig = await config!.getDefault().extension('graphback');
 
   // connect to db
-  const db = await connect(generateConfig.db.dbConfig);
-
-  const schema = makeExecutableSchema({
-    typeDefs,
-    resolvers,
-    resolverValidationOptions: {
-      requireResolversForResolveType: false
-    }
-  });
+  // TODO Embed knex as part of the runtime codegeneration
+  const db = knex({
+    client: generateConfig.dbmigrate.database,
+    connection: generateConfig.dbmigrate.dbConfig,
+  })
 
   const pubSub = new PubSub();
-  const context = createKnexRuntimeContext(db, pubSub);
-  const apolloConfig = {
-    schema,
-    context
-  }
 
-  const apolloServer = new ApolloServer(apolloConfig)
+  const apolloServer = new ApolloServer({
+    typeDefs: loadSchemaFiles(join(__dirname, '/schema/')),
+    resolvers: loadResolversFiles(join(__dirname, '/resolvers/')),
```

Comment:
Getting an error and believe it is something to do with how resolvers are loaded here.
```
Error: "customResolvers" defined in resolvers, but not in schema
```

Reply:
Format of custom resolvers might have changed